tests/driver_hd44780/Makefile

```diff
@@ -0,0 +1,11 @@
+APPLICATION = driver_hd44780
+include ../Makefile.tests_common
+
+BOARD_WHITELIST := arduino-uno
```

Comment:
here you can set all boards that have an arduino compatible pinout and the arduino feature: arduino-duemilanove, arduino-zero, nucleos (64 pins versions).